### PR TITLE
feature: favicon

### PR DESCRIPTION
**Pull request by Cardamced**
Add gorgeous favicon

## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [X] Frontend
- [ ] Backend
- [ ] Other: .....

## General checklist:

- [X] My code follows the style guidelines of this project
- [X] I ran `bun run format`/`bun run lint` before commit
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation where needed
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] I have updated `develop` and merged to my branch before submitting pull request
- [X] My pull request generate no conflicts with `develop` branch
- [X] I requested code review from other team members

## Frontend checklist:

- [X] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [X] My `Javascript` generate no new console errors
- [X] I tested my code cross browsers
- [X] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [X] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by No3ml3 — DISMISSED** [review]

**Review by jaalazard — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
   },
   "dependencies": {
     "cookie-parser": "^1.4.6",
+    "react-favicon": "^2.0.3",
```
Comment (YDrogen) [inline package.json]: Ce package ne sert à rien

File: packages/frontend/web/index.html [hunk 2]
```diff
@@ -4,8 +4,15 @@
...
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>YummyCorn</title>
+    <link
+      rel="icon"
+      href="/yummycorn-favicon.png"
+      sizes="96x68"
+      type="image/svg+xml"
+    />
```
Comment (YDrogen) [inline packages/frontend/web/index.html:10]: Il faut remplacer le favicon existant deux lignes au dessus

File: tsconfig.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
     "allowJs": true,
-    "types": ["bun-types"],
+    "types": ["bun-types"]
   },
-  "extends": "expo/tsconfig.base",
+  "extends": "expo/tsconfig.base"
```
Comment (YDrogen) [inline tsconfig.json]: À retirer

File: packages/frontend/web/src/main.tsx [hunk 4]
```diff
@@ -8,6 +9,12 @@ import { BirthProvider } from './contexts/BirthContext.tsx';
 import './globals.css';
 import router from './router.tsx';
 
+const Main = () => {
+  return <Favicon src='/yummycorn-favicon.png' alt='favicon Yummycorn' />;
+};
+
+export default Main;
```
Comment (YDrogen) [inline packages/frontend/web/src/main.tsx]: Pas utile non plus du coup

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: packages/frontend/web/index.html (hunk 2 above)
Comment (Cardamced) [reply packages/frontend/web/index.html:10]: D'acc. Je ne l'avais pas vu.

File: tsconfig.json (hunk 3 above)
Comment (Cardamced) [reply tsconfig.json]: Done.

File: package.json (hunk 1 above)
Comment (Cardamced) [reply package.json]: Bon, la doc que j'avais trouvée me faisait l'installer... mais si ça fonctionne sans, c'était p'têt pas une bonne doc.

File: packages/frontend/web/src/main.tsx (hunk 4 above)
Comment (Cardamced) [reply packages/frontend/web/src/main.tsx]: C'est enlevé.

**Review by jaalazard — DISMISSED** [review]

**Review by Justicesvr — DISMISSED** [review]

**Review by YDrogen — DISMISSED** [review]

**Review by jaalazard — APPROVED** [review]

**Review by Justicesvr — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**